src/CMakeLists.txt

```diff
@@ -1769,8 +1769,7 @@ install(
   INCLUDES
   DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}")
 
-install(DIRECTORY "${CMAKE_SOURCE_DIR}/data"
-        DESTINATION "${CMAKE_INSTALL_FULL_DATAROOTDIR}/${PROJECT_NAME}")
+install(DIRECTORY "${CMAKE_SOURCE_DIR}/data/" DESTINATION "${CP2K_DATA_DIR}")
```

Comment:
I think, this will copy the files to `${CMAKE_INSTALL_FULL_DATAROOTDIR}/cp2k/data/data`.

Reply:
The final `/` on the `DIRECTORY` argument tells cmake to replace the destination with that directory.  It works fine for me.